tests/UnifiedSpecTests/UnifiedSpecTest.php

```diff
@@ -16,7 +16,6 @@
  * Unified test format spec tests.
  *
  * @see https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst
- * @group serverless
```

Comment:
I was considering doing this while porting tests over the past few weeks. Looks like a good change.

I did overlook the conflict with the `$out` tests and serverless. I do think it'd be reasonable to go back and add `serverless: forbid` to that spec test's `runOnRequirements`.

Reply:
Opened https://github.com/mongodb/specifications/pull/1550 and https://github.com/mongodb/mongo-php-library/pull/1254 to modify the spec tests.